packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env node
+
+/**
+ * automatically creates a module for any CloudFormation namespaces that do not
+ * have an AWS construct library.
+ */
+
+import child_process = require('child_process');
+import fs = require('fs');
+import path = require('path');
+import { promisify } from 'util';
+import cfnspec = require('../lib');
+
+const exist = promisify(fs.exists);
```

Comment:
Could have used `fs-extra` here... Would have made those unnecessary. But optional of course.

Reply:
Done, you are right.